[AmazonMQ] RabbitMQ - Add overview dashboard

agithomas opened the pull request:
- Enhancement

## Proposed commit message

Add overview dashboard for RabbitMQ in ActiveMQ

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [x] Integration testing
- [x] Verified that the ActiveMQ related broker / queue / topic details are not appearing in the dashboard 

## How to test this PR locally

- `elastic-package build`
- `elastic-package stack up -v -d --services package-registry`

## Screenshots


![amazonmq-rabbitmq-overview](https://github.com/user-attachments/assets/e6433e70-aa3f-494e-aeb7-39ca06497444)

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23157)
* Commit: fd598810982b5fe4602e01d7f9d93e7dc6a91f3e

cc @agithomas

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23157","number":23157,"commit":"fd598810982b5fe4602e01d7f9d93e7dc6a91f3e"}],"number":23157}
buildkite-pr-comment-->

daniela-elastic reviewed (commented):
A few points based on the 'Observability integrations dashboards best practices' [guidelines](https://docs.google.com/document/d/1aTqkl0BSYPa1WeKXkKCChQaIqM0J5w4vY-8flHDUj8A/edit?tab=t.0#heading=h.qvg40bcfmooa):
1. Use hyphen "-" to separate complex  widget names that are broken down by certain entity such as "by node", "by broker", etc. 
2. Remove the word "count" or other similar synonyms where it seems redundant. For example, instead of "Queue count", replace with "Queues"; instead of "Total messages count", replace by "Total messages"; instead of "Messages ready count", replace with "Messages ready" or maybe even "Ready messages"; instead of "Consumer count", replace with "Consumers",  and so on and so forth. 
3. Name each section of the dashboard. The first section is unnamed (it contains resource utilization metrics and a melange of connections, consumers, messages and rates), in addition to two named sections for Node metrics and Queue metrics. 
4.

muthu-mps reviewed (commented):
- Can we expand the control section in such a way that it fits the screen width?

  <img width="1124" alt="Screenshot 2025-03-14 at 12 53 45 PM" src="https://github.com/user-attachments/assets/17abc85d-e3e5-4f2e-a4ab-9727d4d5bbfc" />

- Change the title and corresponding labels from `Un-acknowledged` to `Unacknowledged`. 
- And for `Acknowledgment`, Can we update to `Acknowledgement`? As both are correct by making this change it becomes more aligned with the field descriptions. WDYT?
https://github.com/elastic/integrations/blob/f50b59410e4096476673ff1fb92f6075c341ba6b/packages/aws_mq/data_stream/rabbitmq_metrics/fields/fields.yml#L172

agithomas commented:
Thanks, @muthu-mps for the feedback, I have addressed the suggestions. 
The width of the control section I would prefer to keep it as it is - in a high-resolution display when extending to full width, it leads to accessibility issues.

muthu-mps commented:
> Thanks, @muthu-mps for the feedback, I have addressed the suggestions. The width of the control section I would prefer to keep it as it is - in a high-resolution display when extending to full width, it leads to accessibility issues.

The right axis label for `Total messages Vs Ready messages Vs unacknowledged` messages needs an update.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23396)
* Commit: 1b6b605696af6c3af7f5072bb92408ea0f38d6d1

### History
* :green_heart: [Build #23387](https://buildkite.com/elastic/integrations/builds/23387) succeeded 6d6323a38b6a6ce1b6ad3add8b356ae4aee60ab4
* :green_heart: [Build #23157](https://buildkite.com/elastic/integrations/builds/23157) succeeded fd598810982b5fe4602e01d7f9d93e7dc6a91f3e

cc @agithomas

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23396","number":23396,"commit":"1b6b605696af6c3af7f5072bb92408ea0f38d6d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23387","number":23387,"commit":"6d6323a38b6a6ce1b6ad3add8b356ae4aee60ab4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23157","number":23157,"commit":"fd598810982b5fe4602e01d7f9d93e7dc6a91f3e"}],"number":23396}
buildkite-pr-comment-->

agithomas commented:
> The right axis label for Total messages Vs Ready messages Vs unacknowledged messages needs an update.

Great catch. Addressed.

muthu-mps reviewed (approved):
LGTM!
